tests/categorical_test.py

```diff
@@ -41,27 +41,33 @@ def _getRandomizedCategorical(self) -> ak.Categorical:
     def testBaseCategorical(self):
         cat = self._getCategorical()
 
-        self.assertTrue((ak.array([7, 5, 9, 8, 2, 1, 4, 0, 3, 6]) == cat.codes).all())
-        self.assertTrue((ak.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9]) == cat.segments).all())
-        self.assertTrue(
-            (
-                ak.array(
-                    [
-                        "string 8",
-                        "string 6",
-                        "string 5",
-                        "string 9",
-                        "string 7",
-                        "string 2",
-                        "string 10",
-                        "string 1",
-                        "string 4",
-                        "string 3",
-                        "N/A",
-                    ]
-                )
-                == cat.categories
-            ).all()
+        self.assertListEqual(
+            ak.array([7, 5, 9, 8, 2, 1, 4, 0, 3, 6]).to_ndarray().tolist(),
+            cat.codes.to_ndarray().tolist(),
+        )
+        self.assertListEqual(
+            ak.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9]).to_ndarray().tolist(),
+            cat.segments.to_ndarray().tolist(),
+        )
+        self.assertListEqual(
+            ak.array(
+                [
+                    "string 8",
+                    "string 6",
+                    "string 5",
+                    "string 9",
+                    "string 7",
+                    "string 2",
+                    "string 10",
+                    "string 1",
+                    "string 4",
+                    "string 3",
+                    "N/A",
+                ]
+            )
+            .to_ndarray()
+            .tolist(),
```

Comment:
We could simplify this by defining this as a list to begin with instead of using `ak.array()` then using `.to_ndarray().tolist()` to translate it.

Reply:
good suggestion! I'll update this (here and in the other spots)

Reply:
done!